### PR TITLE
Feat: add audio files to pondcast rss feed

### PR DESCRIPTION
**Pull request by maybeanerd**
This PR aims to add the audio files of the pondcast episodes to its feed.

My IDE wanted a few formatting changes, but I limited them to a single commit. If you don't want them in here, I'll revert it.

It's currently a draft as I still need to test, and there's two issues I think I need to solve.

1. Ability to decide between m4a and mp3 - link to the correct one 
2. Provide byte size/file length 

I already have some idea on how to solve this, but wanted you to be able to already provide feedback/opinions, which is why I'm opening this as draft.

The RSS spec for enclosures is the following:
https://www.rssboard.org/rss-specification#ltenclosuregtSubelementOfLtitemgt

---

Solution idea: essentially scrape the pages for potential file paths. During runtime, attempt to load/find the files. Depending on what can be found, load their metadata. 
This would allow to auto detect everything without added manual overhead (e.g. new Inputs in the maker) , and still wouldn't add build processes.
IDK how long it would make the feedMaker take to make the feed, though. Will try it out soonish

### CODE REVIEW

File: pondcast/feed/maker/index.html [hunk 1]
```diff
@@ -120,13 +121,19 @@ <h1>Pondcast feed maker</h1>
     const sortedItems = items.sort((a, b) => a.pubDate - b.pubDate);
     let feedItems = [];
     for (const item of sortedItems) {
+      const itemLink = `http://www.todepond.com/pondcast/${titleToPath(
```
Comment (maybeanerd) [inline pondcast/feed/maker/index.html:124]: The main changes are in the following lines

**Comment by maybeanerd** [comment]
I think this is now something you could review!

An example of the resulting feed is the following, and according to https://validator.w3.org/feed/check.cgi it's valid:

```xml
<?xml version="1.0" encoding="utf-8"?>
      <rss version="2.0">
        <channel>
          <title>Todepond Pondcast</title>
          <link>http://www.todepond.com/pondcast/</link>
          <description>weekly update from the pond</description>
            
        <item>
          <title>Older episodes</title>
          <link>http://www.todepond.com/pondcast/older-episodes/</link>
          <pubDate>Thu, 15 Aug 2024 15:38:00 GMT</pubDate>
          
        </item>
      

        <item>
          <title>Normalise sharing scrappy fiddles</title>
          <link>http://www.todepond.com/pondcast/normalise-sharing-scrappy-fiddles/</link>
          <pubDate>Fri, 16 Aug 2024 15:20:00 GMT</pubDate>
          
        </item>
      

        <item>
          <title>Perfect rhyme</title>
          <link>http://www.todepond.com/pondcast/perfect-rhyme/</link>
          <pubDate>Fri, 16 Aug 2024 15:21:00 GMT</pubDate>
          
        </item>
      

        <item>
          <title>Set</title>
          <link>http://www.todepond.com/pondcast/set/</link>
          <pubDate>Fri, 16 Aug 2024 15:22:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/set/1.m4a" length="12151931" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Multiple angles</title>
          <link>http://www.todepond.com/pondcast/multiple-angles/</link>
          <pubDate>Fri, 16 Aug 2024 15:30:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/multiple-angles/1.m4a" length="14369827" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Wires to wires to wires</title>
          <link>http://www.todepond.com/pondcast/wires-to-wires-to-wires/</link>
          <pubDate>Fri, 16 Aug 2024 15:31:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/wires-to-wires-to-wires/1.m4a" length="16742981" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Infinity rooms</title>
          <link>http://www.todepond.com/pondcast/infinity-rooms/</link>
          <pubDate>Fri, 16 Aug 2024 15:32:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/infinity-rooms/1.m4a" length="13793090" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Replacement</title>
          <link>http://www.todepond.com/pondcast/replacement/</link>
          <pubDate>Fri, 16 Aug 2024 15:33:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/replacement/1.m4a" length="13593679" type="audio/mpeg" />
        </item>
      

        <item>
          <title>CHOOSE: Your hero</title>
          <link>http://www.todepond.com/pondcast/choose-your-hero/</link>
          <pubDate>Fri, 16 Aug 2024 15:34:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/choose-your-hero/1.m4a" length="16243227" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Puss in boots</title>
          <link>http://www.todepond.com/pondcast/puss-in-boots/</link>
          <pubDate>Fri, 16 Aug 2024 15:35:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/puss-in-boots/1.m4a" length="16230011" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Berd has flown away</title>
          <link>http://www.todepond.com/pondcast/berd-has-flown-away/</link>
          <pubDate>Fri, 16 Aug 2024 15:36:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/berd-has-flown-away/1.m4a" length="19075017" type="audio/mpeg" />
        </item>
      

        <item>
          <title>VIDEO VOTE: Tode will wake up</title>
          <link>http://www.todepond.com/pondcast/tode-will-wake-up/</link>
          <pubDate>Fri, 16 Aug 2024 15:37:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/tode-will-wake-up/1.m4a" length="14587586" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Sixteen scrappy fiddles</title>
          <link>http://www.todepond.com/pondcast/sixteen-scrappy-fiddles/</link>
          <pubDate>Fri, 16 Aug 2024 15:38:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/sixteen-scrappy-fiddles/1.m4a" length="15878076" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Pointing to the pieces</title>
          <link>http://www.todepond.com/pondcast/pointing-to-the-pieces/</link>
          <pubDate>Fri, 16 Aug 2024 15:39:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/pointing-to-the-pieces/1.m4a" length="33438070" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Vision pro</title>
          <link>http://www.todepond.com/pondcast/vision-pro/</link>
          <pubDate>Fri, 16 Aug 2024 15:40:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/vision-pro/1.m4a" length="19169389" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Final shot</title>
          <link>http://www.todepond.com/pondcast/final-shot/</link>
          <pubDate>Fri, 16 Aug 2024 15:41:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/final-shot/1.m4a" length="7208137" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Finding ninety nine sands</title>
          <link>http://www.todepond.com/pondcast/finding-ninety-nine-sands/</link>
          <pubDate>Fri, 16 Aug 2024 15:42:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/finding-ninety-nine-sands/1.m4a" length="19563054" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Twenty minute video</title>
          <link>http://www.todepond.com/pondcast/twenty-minute-video/</link>
          <pubDate>Fri, 16 Aug 2024 15:43:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/twenty-minute-video/1.m4a" length="29959716" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Don't do maths</title>
          <link>http://www.todepond.com/pondcast/dont-do-maths/</link>
          <pubDate>Fri, 16 Aug 2024 15:44:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/dont-do-maths/1.m4a" length="19121266" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Why we are coders</title>
          <link>http://www.todepond.com/pondcast/why-we-are-coders/</link>
          <pubDate>Fri, 16 Aug 2024 15:45:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/why-we-are-coders/1.mp3" length="15357120" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Natural code</title>
          <link>http://www.todepond.com/pondcast/natural-code/</link>
          <pubDate>Fri, 16 Aug 2024 15:46:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/natural-code/1.m4a" length="23370368" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Torn leaf zero</title>
          <link>http://www.todepond.com/pondcast/torn-leaf-zero/</link>
          <pubDate>Fri, 16 Aug 2024 15:47:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/torn-leaf-zero/1.m4a" length="29609930" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Elephant in the room + Step by step</title>
          <link>http://www.todepond.com/pondcast/elephant-in-the-room-and-step-by-step/</link>
          <pubDate>Fri, 16 Aug 2024 15:48:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/elephant-in-the-room-and-step-by-step/1.m4a" length="53032344" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Out of time</title>
          <link>http://www.todepond.com/pondcast/out-of-time/</link>
          <pubDate>Fri, 16 Aug 2024 15:49:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/out-of-time/1.m4a" length="27932592" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Confession</title>
          <link>http://www.todepond.com/pondcast/confession/</link>
          <pubDate>Fri, 16 Aug 2024 15:50:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/confession/1.m4a" length="17908176" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Todepond</title>
          <link>http://www.todepond.com/pondcast/todepond/</link>
          <pubDate>Fri, 16 Aug 2024 15:51:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/todepond/1.m4a" length="28807480" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Stitch</title>
          <link>http://www.todepond.com/pondcast/stitch/</link>
          <pubDate>Fri, 16 Aug 2024 15:52:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/stitch/1.m4a" length="34566176" type="audio/mpeg" />
        </item>
      

        <item>
          <title>This is a shepherd</title>
          <link>http://www.todepond.com/pondcast/this-is-a-shepherd/</link>
          <pubDate>Fri, 16 Aug 2024 15:53:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/this-is-a-shepherd/1.m4a" length="29655325" type="audio/mpeg" />
        </item>
      

        <item>
          <title>This is not a fish</title>
          <link>http://www.todepond.com/pondcast/this-is-not-a-fish/</link>
          <pubDate>Fri, 16 Aug 2024 15:54:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/this-is-not-a-fish/1.mp3" length="10754807" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Reaction reaction</title>
          <link>http://www.todepond.com/pondcast/reaction-reaction/</link>
          <pubDate>Fri, 16 Aug 2024 15:55:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/reaction-reaction/1.mp3" length="33012480" type="audio/mpeg" />
        </item>
      

        <item>
          <title>With friends</title>
          <link>http://www.todepond.com/pondcast/with-friends/</link>
          <pubDate>Fri, 16 Aug 2024 15:56:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/with-friends/1.m4a" length="42562455" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Three and a half weeks</title>
          <link>http://www.todepond.com/pondcast/three-and-a-half-weeks/</link>
          <pubDate>Fri, 16 Aug 2024 15:58:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/three-and-a-half-weeks/1.mp3" length="14192352" type="audio/mpeg" />
        </item>
      

        <item>
          <title>Tease open</title>
          <link>http://www.todepond.com/pondcast/tease-open/</link>
          <pubDate>Fri, 16 Aug 2024 15:59:00 GMT</pubDate>
          <enclosure url="http://www.todepond.com/pondcast/tease-open/1.mp3" length="9573888" type="audio/mpeg" />
        </item>
      
        </channel>
      </rss>
```

**Comment by maybeanerd** [comment]
Did some final testing, which is why I added a few more changes.

Needed to add build&copy seperately, as with slow Internet (or without any cache) the scanning of files takes long enough for the browser to not allow copy to clipboard anymore (because of the missing user interaction)

But, I also deployed it to my GitHub pages and it works great now.

**Comment by TodePond** [comment]
did it work? :)

**Comment by maybeanerd** [comment]
It worked!!!
Very nice
There's still some meta info like image and author name missing, but this finally allows audiobookshelf to pull the episodes from the feed. Awesome!

**Pull request merged**